javalink/loader.py

```diff
@@ -66,9 +66,8 @@ def load(self, name):
                     msg = "Wanted class '{}', but '{}' was loaded"
                     raise ValueError(msg.format(name, clazz))
 
-                classes = self.packages.setdefault(package, {})
-                classes[class_name] = clazz
```

Comment:
Now that this happens in all cases, please combine the two `if` statements on L64 and L65:

```python
if clazz and (clazz.package != package or clazz.name != class_name):
```